dist/jarallax.js

```diff
@@ -406,6 +461,7 @@ var Jarallax = function () {
             imgRepeat: 'no-repeat', // supported only for background, not for <img> tag
             keepImg: false, // keep <img> tag in it's default place
             elementInViewport: null,
+            scrollableContainer: null,
```

Comment:
We should update the `README.md` with this new option and explain it's purpose / usage.